documentation/code-gen/configure-typescript-sdk.md

```diff
@@ -0,0 +1,235 @@
+# Readme Configuration Guide for Typescript SDK
```

Comment:
I would elaborate this name a bit for clarification

Azure SDK for Javascript (Node.js and browser) = TypeScript SDK

We may want to be explicit about this to not confuse people with the deprecated node.js SDK repo